python/rascaline/examples/long-range-descriptor.py

```diff
@@ -0,0 +1,427 @@
+ax.set_xlabel("Å")
+ax.set_ylabel("Å")
+
+fig.show()
+
+
+# %%
+# As you can see, the ``cutoff`` radii of the two molecules are completely disjoint.
+# Therefore, a short-range model will not able to describe the intermolecular
+# interactions between our two molecules. To verify we now again create a long-range
+# only descriptor for this system. We use the already defined
+# ``real_space_expansion_long_range`` and ``fourier_space_expansion_long_range``
+
+
+real_space_expansion_long_range = real_space_calculator.compute(atoms_long_range)
+fourier_space_expansion_long_range = fourier_space_calculator.compute(atoms_long_range)
+
+# %%
+# We now firdt verify that the contribution from the short-range descriptors is the same
+# as for a single water molecule. Exemplarily, we compare only the first (Hydrogen)
+# block of each tensor.
+
+
+print("Single water real space spherical expansion")
+print(np.round(real_space_expansion[1].values, 3))
+
+print("\nTwo water real space spherical expansion")
+print(np.round(real_space_expansion_long_range[1].values, 3))
+
+# %%
+# Since the values of the block are the same, we can conclude that there is no
+# information shared between the two molecules and that the short-range descriptor is
+# not able to distinguish the system with only one or two water molecules. Note that the
+# different number of `samples` in ``real_space_expansion_long_range`` reflects the fact
+# that the second system has more atoms then the first.
+#
+# As above, we construct a long-range only descriptor and densify the result for
+# plotting the values.
+
+
+subtracted_expansion_long_range = (
+    fourier_space_expansion_long_range - real_space_expansion_long_range
+)
+
+fourier_space_expansion_long_range = densify_tensormap(
+    fourier_space_expansion_long_range
+)
+subtracted_expansion_long_range = densify_tensormap(subtracted_expansion_long_range)
+
+
+# %%
+# As above, we plot the values of the spherical expansions for the Fourier and the
+# subtracted (long range only) spherical expansion.
+
+
+for key in fourier_space_expansion.keys:
+    fig, ax = plt.subplots(2, layout="constrained")
+
+    values_subtracted = subtracted_expansion_long_range[key].values
+    values_fourier_space = fourier_space_expansion_long_range[key].values
+
+    # maximal and minimal values for same scaling of axes
+    vmin = min(np.min(values_subtracted), np.min(values_fourier_space))
+    vmax = max(np.max(values_subtracted), np.max(values_fourier_space))
+
+    ax[0].set_title(f"center_type={key.values[0]}\n Fourier space sph. expansion")
+    im = ax[0].matshow(values_fourier_space, vmin=vmin, vmax=vmax)
+    ax[0].set_ylabel("sample index")
+
+    ax[1].set_title("Difference between Fourier and real space sph. expansion")
+    ax[1].matshow(values_subtracted, vmin=vmin, vmax=vmax)
+    ax[1].set_ylabel("sample index")
+    ax[1].set_xlabel("property index")
+
+    fig.colorbar(im, ax=ax[0], orientation="horizontal", fraction=0.1, label="values")
```

Comment:
this plot looks weird in the rendered documentation, the left & right side have different width. Is this intentional?

Reply:
No, idea what sphinx-gallery and matplotlib are doing here. I tried resizing the figures but it didn't help. I am now showing them in different "cells".